game-core/src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -69,22 +67,35 @@
   private final Collection<SAXParseException> errorsSax = new ArrayList<>();
   public static final String DTD_FILE_NAME = "game.dtd";
   private final String mapName;
+  private final boolean noValidation;
```

Comment:
I can't think of anything better, so I think this will work fine.  My only recommendation would be to possibly use an enum instead of a Boolean here.  Simply because when you pass `false` for a Boolean that represents a negative, you have to stop and think what you're actually doing.  Having an enum with elements `VALIDATION` and `NO_VALIDATION` would be clearer.

Or just reversing the meaning of the field might work, too, if you think the enum is overkill. :smile: